src/environments/environment.ts

```diff
@@ -22,7 +22,7 @@ export const environment = {
   PATTERN_ATLAS_UI_URL:
     window['env']['PATTERN_ATLAS_HOST_NAME'] &&
     window['env']['PATTERN_ATLAS_PORT']
-      ? `http://${window['env']['PATTERN_ATLAS_HOST_NAME']}:${window['env']['PATTERN_ATLAS_UI_PORT']}/patternatlas`
+      ? `http://${window['env']['PATTERN_ATLAS_HOST_NAME']}:${window['env']['PATTERN_ATLAS_UI_PORT']}`
```

Comment:
should /patternatlas be added, also at line 26, @lharzenetter?

Reply:
@manuwei ?

Reply:
the link is correct, since the patternatlas ui is currently running at '/'. changing it would require to adapt the patternatlas ui first